source/iterators.tex

```diff
@@ -1543,7 +1543,7 @@
 \pnum
 The \libconcept{sized_sentinel_for} concept specifies
 requirements on an \libconcept{input_or_output_iterator} and
-a corresponding \libconcept{sentinel_for}
+a corresponding \libconcept{sentinel}
```

Comment:
This doesn't look right; there is no concept "sentinel", it seems.

Reply:
Ah, right.